keyboards/zeta/rev1/rules.mk

```diff
@@ -0,0 +1,30 @@
+
+OPT_DEFS += -DZETA_PROMICRO
+
+
+# # project specific files
+SRC = matrix.c
+MCU = atmega32u4
+
+BOOTLOADER = caterina
+
+# Build Options
+#   comment out to disable the options.
+
+BOOTMAGIC_ENABLE = lite   # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes     # Mouse keys
+EXTRAKEY_ENABLE = yes     # Audio control and System control
+CONSOLE_ENABLE = no       # Console for debug
+COMMAND_ENABLE = yes      # Commands for debug and configuration
+CUSTOM_MATRIX = yes       # Remote matrix from the wireless bridge
```

Comment:
Ideally, we want to use lite, as that handles debounce and some other stuff for us. 
https://docs.qmk.fm/#/custom_matrix?id=custom-matrix

Reply:
@drashna would changing to lite eliminate the need for these suggested changes?

Reply:
If you need help with that, don't hesitate to ask. 

If you need a reference on how, here is the work I did for the ergodox EZ:
https://github.com/qmk/qmk_firmware/commit/11c308d436180974b7719ce78cdffdd83a1302c0#diff-be76b2c6078a2837fdee9d715dbafd5eb34c26fd5ea0911bdf0a942b3b0c5013n

Reply:
Thank you **very** much for that commit link, @drashna. That will help considerably. It's going to take me some time to get this done. If I end up sitting on this for more than a week, I'll put this PR on hold until I can get it done.

Reply:
You're very welcome!  
And yeah, sometimes it helps to have a frame of reference. At least, I know it does for me.